.travis.yml

```
@@ -6,5 +6,6 @@ services:
 script:
   - sbt compile
   - sbt test
+  - sbt gatling:test
```

what tests does it play?

---

It launch the ImapAuthenticationSimulation and ImapStoreSimulation.
I thought that it can be useful to add them, to fail fast before launching the integration test.
What do you think of it ? If we remove it we can remove 'sbt test' too which really do nothing.

---

you can't run a simulation without a working server. Maybe I miss something